include/FixedPoint.h

```diff
@@ -26,6 +26,7 @@ class FixedPoint
 
     FixedPoint& operator-=(const FixedPoint& b) { m_value -= b.m_value; return *this; }
     const FixedPoint operator-(const FixedPoint& b) const { FixedPoint result = *this; result -= b; return result; }
+    FixedPoint& operator-() { m_value *= -1 ; return *this; }
```

Comment:
Remove extra space between -1 and ;

Reply:
It is completely wrong by the way :D

Should be something like:
FixedPoint operator-() { FixedPoint negative(*this); negative.m_value *= -1; return negative; }